metis-enrichment/metis-enrichment-client/src/test/java/eu/europeana/enrichment/rest/client/EnrichmentWorkerTest.java

```diff
@@ -83,13 +83,23 @@
   
   @Test
   public void testEnrichmentWorkerHappyFlow() throws DereferenceOrEnrichException {
-    for (Mode mode : Mode.values()) {
-      testEnrichmentWorkerHappyFlow(mode);
-    }
+	  Logger logger = LogManager.getLogger(EnrichmentWorker.class);
+	  Level origLevel = logger.getLevel();
+	  
+	  Configurator.setLevel(logger.getName(), Level.DEBUG);
+	  for (Mode mode : Mode.values()) {
+		  testEnrichmentWorkerHappyFlow(mode);
+	  }
+	  
+	  Configurator.setLevel(logger.getName(), Level.INFO);
+	  for (Mode mode : Mode.values()) {
+		  testEnrichmentWorkerHappyFlow(mode);
+	  }
+	  	  
+	  Configurator.setLevel(logger.getName(), origLevel);
   }
```

Comment:
I don't think we should do the exact same tests for different log levels unless we also check the logging behavior somewhere in this method. This means prescribing what logging should take place at different levels. 

However, I think that this goes a bit too far and the method should be reverted to it's original state. Any thoughts?

Reply:
EnrichmentWorker writes logs specifically based on the set logging level. My main goal here was to increase code coverage by testing for all used log levels (debug and info in this case).

Log4J2 allows this during runtime using the Configurator.setLevel method and I have not been able to get runtime logging level changes to work using pure SLF4J.

Reply:
Well the coverage gain must be practically nothing... negligible indeed, so to me not worth the trouble. And if you just want the coverage, why not just test it with the DEBUG level? Why also with INFO? On top of all this, I think it is not the right way to increase the coverage anyway as we are not actually testing the extra code that is covered. Lastly, I think it certainly is not a valid reason for such a drastic move as switching logging frameworks. 

For all these reasons I would still propose that this code be reverted to the original state.

Reply:
Done.